lib/intercom/api_operations/find_all.rb

```diff
@@ -5,7 +5,7 @@ module Intercom
   module ApiOperations
     module FindAll
 
-      def find_all(params)
+      def find_all(params={})
```

Comment:
In what scenario would the params not be supplied as a hash here? Passing a default value will simply bypass the bad request error that we're throwing in the next line. 

Could you give a bit of extra context about how this fixes it?

Does passing this default value break it for the other classes that use this module?